cosinnus_event/models.py

```diff
@@ -705,10 +706,15 @@ class ConferenceEvent(Event):
         help_text='If an event is a break, no rooms will be created for it, and it will be displayed differently',
         default=False)
 
+    # Checkbox for visible / hidden mode of the particular event
+    is_visible = models.BooleanField(_('Event is visible'), 
```

Comment:
Please rename the field so it is clearer what it does. Something like `is_event_visible_on_microsite` or similar. `is_visible` is confusing as it implies that the event itself would be made visible/invisible with this switch.